core/controllers/collection_viewer.py

```diff
@@ -80,10 +80,12 @@ def get(self, collection_id):
 class CollectionDataHandler(base.BaseHandler):
     """Provides the data for a single collection."""
 
+    @require_collection_playable
```

Comment:
Since that decorator is used for the collection player itself, I think this is perfectly reasonable. Good catch.